code-with-comment.html

```diff
@@ -44,31 +46,48 @@
           type: Number,
           notify: true
         },
-        indexItem: Number,
         hideAddComments: {
           type: Boolean,
           value: true
         },
-        hideComments: {
-          type: Boolean,
-          value: false
-        },
-        comments: Array
+        startLine: Number,
+        comments: Array,
+        user: Object,
```

Comment:
The user is needed for the pushing of comments.
I'll look at this in the next pr to include with the callback.